vignettes/articles/kobo-asset-submissions-data.yml

```diff
@@ -0,0 +1 @@
+
```

Comment:
This cassette should be non-empty.  See my comment on line 33 of vignettes/articles/wrangling_survey_responses.Rmd

Reply:
pushed it! also set up the config for vcr properly so that it looks at the articles folder.. i guess before it was doing some dark magic on my pc to find the stuff in the tests/fixtures folder somehow.. can you check again?